personhood/contracts/contract_spawner.go

```diff
@@ -154,7 +154,7 @@ func (c *ContractSpawner) Spawn(rst byzcoin.ReadOnlyStateTrie, inst byzcoin.Inst
 			if arg.Name == "coinValue" {
 				if len(arg.Value) < 8 {
 					return nil, nil,
-						errors.New("getCoin needs to have a value of 8 bytes")
+						errors.New("GetCoin needs to have a value of 8 bytes")
```

Comment:
[No caps in error messages](https://staticcheck.io/docs/checks#ST1005)

```suggestion
						errors.New("needs to have a value of 8 bytes")
```